standard-bridge-custom-token/README.md

```diff
@@ -11,22 +11,32 @@ For an L1/L2 token pair to work on the Standard Bridge the L2 token contract mus
 
 ## Customizing the `L2StandardERC20` implementation
 
-Our example here implements a custom token [`L2CustomERC20`](contracts/L2CustomERC20.sol) based on the `L2StandardERC20` but with `8` decimal points, rather than `18`.
+Our example here implements a custom token 
+[`L2CustomERC20`](contracts/L2CustomERC20.sol) based on the `L2StandardERC20` but which mints
+the deployer 10,000 tokens when it is deployed.
```

Comment:
There is an issue with new lines being inserted in the text all over the place. I've noticed this not just for tutorials but for all text your editor touches @qbzzt could this be a local setting of yours that propagates?

Reply:
It's something I do fairly automatically, because it makes life easier if I ever have to use a low-tech editor such as nano. Is there a problem with this practice?

Reply:
There is a problem when reviewing changes as there's lots of whitespace changes that are polluting the diff view I use to read. Either we all use the same editor, i.e. we reformat all text to comply with what you need or we switch it off. Either way I'm happy with looking at more meaningful diff.

Reply:
OK, since it makes a problem for you I'll undo it.